packages/ui/components/Tabs/index.js

```diff
@@ -0,0 +1,144 @@
+import React, { useRef, useState, useEffect, useCallback } from 'react'
+import { observer } from 'startupjs'
+import propTypes from 'prop-types'
+import { FlatList } from 'react-native'
+import Div from './../Div'
+import { TabsProvider } from './tabsContext'
+import Tab from './Tab'
+import './index.styl'
+
+function Tabs ({
+  containerStyle,
+  children,
+  iconPosition,
+  activeStyle,
+  style,
+  onChange,
+  value
+}) {
+  const values = []
+
+  React.Children.toArray(children).forEach((child, index) => {
+    const childValue = child.props.value === undefined ? index : child.props.value
+    values.push({ [childValue]: index })
+  })
+
+  const indexOfValue = key => {
+    return value && values.find(obj => Object.keys(obj)[0] === key)[key]
+  }
```

Comment:
```
const valueToIndex = useMemo(() => {
  const _valueToIndex= {}

  React.Children.toArray(children).forEach((child, index) => {
    _valueToIndex[child.props.value || index] = index 
  }

  return _valueToIndex
}, [React.Children.count(children)])
```

add 
remove `actualTab`, and use `const tabIndex = valueToIndex[value] || 0` everywhere

Reply:
done